services/github/src/handlers.js

```diff
@@ -256,6 +282,7 @@ class Handlers {
       commit_sha: sha,
       body,
     });
+    this.markCommentSent(organization, repository, sha, error, pullNumber);
```

Comment:
I wondered if ``pullNumber`` should be sent in this call, since we know it is falsey, but I think we need to do it to match the call to ``isDuplicateComment``, so 👍